src/utils.js

```diff
@@ -535,6 +535,30 @@ function serializeKeysForMessage(keys, maxLength) {
   return '';
 }
 
+function sanitize(input, sanitizeKeys) {
```

Comment:
how does this deal with cycles? or will the data be guaranteed by this point not to have any? how strong is that guarantee? (afraid of freezing a browser in some weird scenario)

Reply:
It didn't. Thanks for catching that!